src/PaymentGateways/PayPalCommerce/onBoardingRedirectHandler.php

```diff
@@ -364,12 +364,21 @@ private function redirectWhenOnBoardingFail( $errorMessage ) {
 	 */
 	private function registerPayPalSSLNotice() {
 		if ( is_ssl() && empty( $this->webhooksRepository->getWebhookConfig() ) ) {
+			$logLink = sprintf(
+				'<a href="%1$s">%2$s</a>',
+				admin_url( '/edit.php?post_type=give_forms&page=give-tools&tab=logs' ),
+				esc_html__( 'check logs', 'give' )
+			);
+
 			Give_Admin_Settings::add_error(
 				'paypal-webhook-error',
-				esc_html__(
-					'There was a problem creating a webhook for your account. Please try disconnecting and then
-					reconnect. If the problem persists, please contact support',
-					'give'
+				sprintf(
+					esc_html__(
+						'There was a problem creating a webhook for your account. Please try disconnecting and then
+					reconnect. If the problem persists, %1$s and please contact support',
+						'give'
+					),
+					$logLink
```

Comment:
It's more important that we link them to the PayPal connect page than the logs. Users won't know what to do with the logs.

Reply:
@JasonTheAdams When the admin will get this notice, he will be on PayPal connect page. This error notice only shows the admin PayPal onboarding redirect page. So PayPal connects page link does not make sense. Let me know what do you think.

Reply:
I see. I guess the biggest reason we felt the need for the link before is because there was something wrong with the token, so it was showing up on other pages. Ok, forget the link then and change the wording to:

> There was a problem setting up the webhooks for your PayPal account. Please try disconnecting and reconnecting your PayPal account. If the problem persists, please contact support and provide them with the latest [logs data].

Reply:
fixed: https://github.com/impress-org/givewp/pull/5389/commits/0c90193cee8cb966f66847ae3341d59d6186019f

![image](https://user-images.githubusercontent.com/1784821/96746931-50dc8580-13e5-11eb-8d66-c680249ab4fe.png)